pkg/microservice/aslan/core/common/service/registry.go

```diff
@@ -67,6 +73,15 @@ func GetDefaultRegistryNamespace(log *zap.SugaredLogger) (*models.RegistryNamesp
 	return resp, nil
 }
 
+func GetRegistryNamespace(registryID string, log *zap.SugaredLogger) (*models.RegistryNamespace, error) {
+	resp, err := mongodb.NewRegistryNamespaceColl().Find(&mongodb.FindRegOps{ID: registryID})
+	if err != nil {
+		log.Errorf("get default registry error: %v", err)
+		return resp, fmt.Errorf("get default registry error: %v", err)
```

Comment:
seems like this function returns registry namespace with particular id, not default registry